lib/skate/application.ex

```diff
@@ -9,12 +9,21 @@ defmodule Skate.Application do
   def start(_type, _args) do
     load_runtime_config()
 
+    start_data_processes? = Application.get_env(:skate, :start_data_processes)
+
     # List all child processes to be supervised
     children =
-      [{Skate.Repo, []}, Skate.WarmUp] ++
-        if Application.get_env(:skate, :start_data_processes) do
+      [{Skate.Repo, []}] ++
+        if start_data_processes? do
+          [
+            Schedule.Supervisor
+          ]
+        else
+          []
+        end ++
```

Comment:
question(non-blocking): is there a reason to start this after `Skate.Repo` instead of before?

Reply:
I'm pretty sure that `Skate.Repo` doesn't block on `init/1`, so I don't think it matters, I am just curious.

Reply:
I think it's better to leave it first because that way it has more time to spin up its connection pool in the background while other things happen.

Reply:
Which in turn means that hopefully by the time we get to launching `WarmUp` it's more likely that the connection pool is in a good state.